onnxruntime/core/graph/graph_utils.h

```diff
@@ -14,12 +14,15 @@ bool IsSupportedOptypeVersionAndDomain(const Node& node,
                                        ONNX_NAMESPACE::OperatorSetVersion version,
                                        const std::string& domain = kOnnxDomainAlias);
 
-Status ForAllMutableSubgraphs(Graph& main_graph, std::function<Status(Graph&)> func);
-Status ForAllSubgraphs(const Graph& main_graph, std::function<Status(const Graph&)> func);
-
 /** Check whether the node has a single input and a single output. */
 bool IsSingleInSingleOutNode(const Node& node);
 
+/** Returns true if the graph has the given input.*/
+bool HasGraphInput(const Graph& graph, const NodeArg* input);
+
```

Comment:
nit: IsGraphInput may be clearer as we're checking if the NodeArg is a graph input.